layout/pages/zoning/zoning.xml

```diff
@@ -224,12 +224,20 @@
 		</Panel>
 		<Panel id="InfoPanel" class="zoning__menu-section hide">
 			<Panel class="w-full flow-down">
-				<Label id="SelectionMode" class="zoning__property-label" />
+				<Label id="SelectionMode" class="zoning__property-label" text="{s:selectionModeHint}" />
 				<Panel class="zoning-info__binds-list">
 					<Panel class="zoning-info__bind">
 						<Label class="zoning-info__bind-action" text="Select" />
 						<Label class="zoning-info__bind-key" text="MOUSE1" />
 					</Panel>
+					<Panel class="zoning-info__bind">
+						<Label class="zoning-info__bind-action" text="Back" />
+						<Label class="zoning-info__bind-key" text="MOUSE2" />
+					</Panel>
+					<Panel class="zoning-info__bind">
+						<Label class="zoning-info__bind-action" text="Cancel region edit" />
+						<Label class="zoning-info__bind-key" text="ESCAPE" />
+					</Panel>
```

Comment:
Shouldn't this stuff be localized?

Reply:
yep, changed everything but the keybind strings.